router/batchrouter/batchrouter.go

```diff
@@ -60,6 +60,7 @@ var (
 	readPerDestination                 bool
 	disableEgress                      bool
 	toAbortDestinationIDs              string
+	netClientTimeout                   time.Duration
```

Comment:
Maybe we can name it httpTimeout to be consistent.

Reply:
This is the same name we are using in router for time out, Do we change it at both the places?